### ISSUE TITLE
Error with multiline ternary operators

### ISSUE THREAD

**Issue by jrmhaig** [bbatsov/rubocop#4794]
## Expected behavior

Describe here how you expected RuboCop to behave in this particular situation.

With this sample code:

```ruby
one ||
  two ? 3 : 5
```

Rubocop should report the offence with multi-line ternary operators according to the configuration.

## Actual behavior

The offence is reported correctly but with an error:

```
1 error occurred:
An error occurred while Layout/MultilineOperationIndentation cop was inspecting /Users/jrmhaig/workspace/rubocop_test/test.rb:1:0.
Errors are usually caused by RuboCop bugs.
Please, report your problems to RuboCop's issue tracker.
Mention the following information in the issue report:
0.50.0 (using Parser 2.4.0.0, running on ruby 2.4.1 x86_64-darwin16)
Finished in 0.12138300016522408 seconds
```

## Steps to reproduce the problem

Create a file `test.rb` containing:

```ruby
one ||
  two ? 3 : 5
```

and execute Rubocop.

## RuboCop version

Include the output of `rubocop -V`. Here's an example:

```
$ rubocop -V
0.50.0 (using Parser 2.4.0.0, running on ruby 2.4.1 x86_64-darwin16)
```

**Comment by Drenmi** [comment]
Not sure I understand. This: `two ? 3 : 5` is a single line ternary. 🤔

**Comment by rrosenblum** [comment]
I think I have a fix for this. I will try to create a PR soon. I need to go over some other test cases.
